lib/game.ex

```diff
@@ -5,28 +5,29 @@ defmodule Game do
             winner: nil, 
             board: %Board{}.spaces
             
-  # convert to private functions?
-  def check_three([x, x, x]), do: true
-  def check_three([a, b, c]), do: false
+  defp check_three([x, x, x]), do: x
+  defp check_three([_a, _b, _c]), do: false
 
-  # convert to private functions?
-  def check_line([head | tail]), do: [check_three(head) | check_line(tail)]
-  def check_line([]), do: []
+  defp check_line([head | tail]), do: [check_three(head) | check_line(tail)]
+  defp check_line([]), do: []
 
-  # convert to private functions?
-  def convert_to_columns([a, b, c, d, e, f, g, h, i]), do: [[a, d, g], [b, e, h], [c, f, i]]
-  def convert_to_diagonals([a, _b, c, _d, e, _f, g, _h, i]), do: [[a, e, i], [g, e, c]]
-  def convert_to_rows(board), do: Enum.chunk(board, 3)
+  defp convert_to_columns([a, b, c, d, e, f, g, h, i]), do: [[a, d, g], [b, e, h], [c, f, i]]
+  defp convert_to_diagonals([a, _b, c, _d, e, _f, g, _h, i]), do: [[a, e, i], [g, e, c]]
+  defp convert_to_rows(board), do: Enum.chunk(board, 3)
 
-  def game_over?(board) do
+  def get_winner(board) do
+    Enum.find(convert_to_rows(board) |> check_line, fn(x) -> x end) ||
```

Comment:
btw, `fn(x) -> x end` is the same as `&(&1)`.

You can capture _any_ function as a value with the `&` operator, and then use `&N` (`&1`, `&2`, etc) to refer to the arguments of the function.

So, `&(&1)` means "an anonymous function, that returns the first argument".